gulp.d/tasks/build.js

```diff
@@ -51,9 +49,8 @@ module.exports = (src, dest, preview) => () => {
     postcssVar({ preserve: true }),
     preview ? postcssCalc : () => {},
     autoprefixer,
-    preview
-      ? () => {}
-      : (css, result) => cssnano({ preset: 'default' })(css, result).then(() => postcssPseudoElementFixer(css, result)),
+    preview ? () => {} : cssnano({ preset: 'default' }),
```

Comment:
cssnano api changed and a promise is no longer returned therefore, `.then()` isn't working anymore. Broke these out into two calls instead, seemed like the easiest way to fix it